cachito/web/config.py

```diff
@@ -26,6 +26,13 @@ class Config(object):
     CACHITO_USER_REPRESENTATIVES: List[str] = []
     CACHITO_WORKER_USERNAMES: List[str] = []
 
+    # Temp dir used by the Prometheus Flask Exporter to coalesce the metrics from the threads
+    if "PROMETHEUS_MULTIPROC_DIR" not in os.environ.keys():
+        PROMETHEUS_METRICS_TEMP_DIR = tempfile.gettempdir()
+        os.environ["PROMETHEUS_MULTIPROC_DIR"] = PROMETHEUS_METRICS_TEMP_DIR
```

Comment:
https://github.com/prometheus/client_python has some more info about this environment variable:
> The `PROMETHEUS_MULTIPROC_DIR` environment variable must be set to a directory that the client library can use for metrics. This directory must be wiped between process/Gunicorn runs (before startup is recommended).
>
> This environment variable should be set from a start-up shell script, and not directly from Python (otherwise it may not propagate to child processes).

I'm not sure if the "not propagated to child processes" issue could affect us, but setting it using an external mechanism rather than Python does seem cleaner.

I would rather not set the variable here, instead just check that it is already set. We should probably set it in docker-compose for the local deployment, or we could have a default directly in the Dockerfile.